### PR TITLE
docs: update system requirements

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

Updated outdated in the troubleshooting guide. Set a minimal version for Podman.

Note: I chose 4.1 because it is the first version fixing the mentioned issue. It might require an update to a newer version.

### What issues does this PR fix or reference?

refs: #3045

### CODE REVIEW

File: website/docs/troubleshooting.md [hunk 1]
```diff
@@ -12,11 +12,7 @@ If you cannot find your issue here or in the documentation, please fill an issue
 
 #### System Requirements
 
-The tool connects to Podman using the socket on the host on macOS and on a named pipe on Windows.
-This is available only on Podman 4.0.2+
-So, please check your version and update.
-
-On Windows, the named pipe is `//./pipe/docker_engine` when Docker Desktop is not installed. It will be solved by <https://github.com/containers/podman/issues/13502> / <https://github.com/containers/podman/pull/13655>. During that time, you might start Docker Desktop so the named pipe is the one expected.
+- Use Podman version 4.1 or greater.
```
Comment (benoitf) [inline website/docs/troubleshooting.md:15]: you can't remove from one side without adding it at another side I would say

the information `On Windows, the named pipe is `//./pipe/docker_engine` when Docker Desktop is not installed` is really important, so people can check if it exists, etc

Comment (themr0c) [reply website/docs/troubleshooting.md:15]: Do the user need to know and care now that the issue is solved?

**Comment by afbjorklund** [comment]
What are the major blockers for running with Podman v3, the version available on Ubuntu and Debian ?

File: website/docs/troubleshooting.md (hunk 1 above)
Comment (benoitf) [reply website/docs/troubleshooting.md:15]: so as a user you may not care about how the issue is solved but you're interested in knowing which npipe is used

**Comment by benoitf** [comment]
I think there is no blocker for podman v3 as long as you're on Linux and don't use machines. (I would say most of Linux users don't use podman machines)
Most of the required changes have been done on the 'machine side'

File: website/docs/troubleshooting.md (hunk 1 above)
Comment (jeffmaury) [reply website/docs/troubleshooting.md:15]: Agree we should either document if user has Podman >= 4.0.2 and < 4.1 that Docker Desktop should be started (rather that installed) or switch to Podman 4.1+

**Review by jeffmaury — CHANGES_REQUESTED** [review]

**Pull request closed without merging**